test/test_ride.py

```diff
@@ -0,0 +1,185 @@
+import unittest
+from app.models.rides import rides
+
```

Comment:
Are you testing the models on this file??

Reply:
Yes I am.